lib/inspect_unicode.c

```diff
@@ -455,13 +455,17 @@ static int validate_file(const char *fpath, __attribute__((unused)) const struct
          *     rpminspect-1.47.0/lib/magic.c
          */
         localpath += strlen(build);
```

Comment:
Just one style comment here, but can you put a blank line between 'localpath += strlen(build);' and the block you added?